machine-learning/05-machine_learning_exercise.Rmd

```diff
@@ -0,0 +1,245 @@
+```
+
+Now that you have `proteasome_df`, let's join it with our metadata using
+an `inner_join` from `dplyr` to form a new data.frame called `prot_meta_df`.
+We'll join by the sample names, which is what `by = "geo_accession"` specifies.
+
+```{r}
+prot_meta_df <- dplyr::inner_join(x = , 
+                                  y = , 
+                                  by = "geo_accession")
+```
+
+Let's find out what samples have the highest expression values for the
+proteasome latent variable.
+We can sort `prot_meta_df` with `arrange`.
+Use the `Help` panel or `?arrange` to learn what to expect from this function.
+
+```{r}
+prot_meta_df %>%
+```
+
+What do you notice about the samples with the highest values?
+Look up the mechanism of action for the drugs in `treatment`.
+Do these results make sense?
+
+### Line graph
+
+Let's make a plot that shows the change of the proteasome latent variable
+expression level over time, when treated with the drug that comes to the top
+of the list.
+We'll look at a single tissue of your choosing.
+This requires use to do the following:
+
+1. Filter `prot_meta_df` to samples from your tissue of interest and the
+   drug that we are interested in
+2. Make a line graph that tracks the expression value of the proteasome 
+   latent variable during different time points of the experiment.
+
+Save this plot as `p`.
+
+```{r}
+p <- prot_meta_df %>%
+  # use these lines to filter to only the samples that are treated with the drug
+  # of interest and to the tissue of your choice
+  %>% 
+  ggplot(aes(x = ,  # what needs to be on the x-axis?
+             y = ,  # what needs to be on the y-axis?
+             color = cell_line,  # color by individual cell line
+             # we need to use group to make our line plot
+             group = cell_line)) +
+  # use facets to separate dosages
+  facet_wrap(~ dose_nM) +
+  # these two geoms make a line graph connecting individual points
+  geom_point() +
+  geom_line() +
+  # use this scale for accessibility
+  colorblindr::scale_color_OkabeIto()
+```
+
+Let's see what the plot looks like with this chunk below.
+
+```{r}
+p
+```
+
+**Based on what you know about the mechanism of action of this drug, do these 
+results look how you expect?
+What did the PLIER model uncover and what does the plot mean?**
+
+#### Customizing the plot
+
+We can add to `p` or tweak things by using the `+` operator without having to
+repeat the code that set it up.
+In the chunk below, we're changing the theme of the plot to `theme_bw()`.
+
+```{r}
+p + theme_bw()
+```
+
+Using the new theme, label the x-axis Time (Hours) and the y-axis Proteasome 
+LV Expression Level.
+Make the title the name of the drug you are plotting data for.
+
+```{r}
+
+```
+
+If we wanted to change the spacing of the x-axis, such that the time points are
+evenly spaced, we'd have to make `time_hours` into a factor.
+_This is not always an appropriate visualization choice, it depends on your
```

Comment:
Would add a second sentence here noting when it is/isn't an appropriate choice. Presumably when the interval is important, don't make them evenly spaced.